components/feature/session/src/main/res/layout/mozac_feature_choice_group_item.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+          style="?android:attr/listSeparatorTextViewStyle"
+          android:id="@android:id/text1"/>
```

Comment:
Let's prefer using our own Ids?

Reply:
I'm using Android's  String should I also created ours too?
```
android.R.string.cancel
android.R.string.ok
```

Reply:
I updated it to use our strings too!